src/libraries/System.Net.Mail/tests/Unit/System.Net.Mail.Unit.Tests.csproj

```diff
@@ -272,6 +252,29 @@
     <Compile Include="$(CommonPath)Interop\Windows\Interop.UNICODE_STRING.cs"
              Link="Common\Interop\Windows\Interop.UNICODE_STRING.cs" />
   </ItemGroup>
+
+  <!-- Non Browser specific files - internal and security -->
+  <ItemGroup Condition="'$(TargetsBrowser)' != 'true'">
+    <Compile Include="..\..\src\System\Net\Mail\SmtpLoginAuthenticationModule.cs"
+             Link="ProductionCode\SmtpLoginAuthenticationModule.cs" />
+    <Compile Include="..\..\src\System\Net\Mail\SmtpNegotiateAuthenticationModule.cs"
+             Link="ProductionCode\SmtpNegotiateAuthenticationModule.cs" />
+    <Compile Include="..\..\src\System\Net\Mail\SmtpNtlmAuthenticationModule.cs"
+             Link="ProductionCode\SmtpNtlmAuthenticationModule.cs" />
+    <Compile Include="$(CommonPath)System\Net\NTAuthentication.Common.cs"
+             Link="Common\System\Net\NTAuthentication.Common.cs" />
+    <Compile Include="$(CommonPath)System\Net\Security\Unix\SafeFreeCredentials.cs"
+             Link="Common\System\Net\Security\Unix\SafeFreeCredentials.cs" />
+    <Compile Include="$(CommonPath)System\Net\Security\Unix\SafeDeleteContext.cs"
+             Link="Common\System\Net\Security\Unix\SafeDeleteContext.cs" />
```

Comment:
It seems like there are some Unix files here that are included on Windows as the condition is true on Windows as well, which is causing the build to fail.